lib/reference.coffee

```diff
@@ -37,7 +38,9 @@ class Reference
 
     # Split on ":" for chapter and verse. If it's a chapter reference
     # (e.g. John 1) then @verse is undefined
-    [@chapter, @verse] = chapterAndVerse.split(':')
+    parts = chapterAndVerse.split(':')
+    @chapter = +parts[0]
```

Comment:
Could you use something like:

```coffee
[ @chapter, @verse ] = chapterAndVerse.split(':')
@chapter = Number(@chapter)
@verse = Number(@verse) or null
```

I'd like to avoid the `variable = +variable` pattern for type coercion.

Also, any reason why you used `-1` to indicate no verse? I'd prefer to keep it more explicit - either null or undefined.